cg/apps/demultiplex/demultiplex_api.py

```diff
@@ -0,0 +1,112 @@
+"""This api should handle everything around demultiplexing"""
+import logging
+from pathlib import Path
+from typing import Iterable, Optional
+
+from cg.apps.demultiplex.flowcell import Flowcell
+from cg.apps.demultiplex.sbatch import DEMULTIPLEX_COMMAND, DEMULTIPLEX_ERROR
+from cg.apps.slurm.slurm_api import SlurmAPI
+from cg.models.demultiplex.sbatch import SbatchCommand, SbatchError
+from cg.models.slurm.sbatch import Sbatch
+
+LOG = logging.getLogger(__name__)
+
+
+def fetch_flowcell_paths(in_path: Path) -> Iterable[Path]:
+    """Loop over a directory and find all flowcell files
+
+    It is assumed that the flowcell directories are situated as children to in_path
+    """
+    for child in in_path.iterdir():
```

Comment:
rename child to subdir? more technical term

Reply:
lets keep it technical 🎖️